dropwizard-jdbi3-transactions-core/src/main/kotlin/com/campspot/jdbi3/TransactionEventListener.kt

```diff
@@ -37,7 +38,11 @@ class TransactionApplicationListener(private val daoManager: DAOManager) : Appli
       } else if (eventType == RequestEvent.Type.RESP_FILTERS_START) {
         transactionAspect.afterEnd()
       } else if (eventType == RequestEvent.Type.ON_EXCEPTION) {
-        logger.error("exception event", event.exception);
+        if (ignoredExceptionClasses.contains(event.exception.javaClass)) {
```

Comment:
for super frequent/expected/noisy exceptions

Reply:
That `ignoredExceptionClasses` list is going to be so nice for avoiding extra noise 👏